manual/de/ch_Finance-Quote.xml

```diff
@@ -281,9 +283,7 @@ gnucash-cli --quotes get <replaceable>${HOME}/gnucash-filename</replaceable> &gt
       </step>
     </procedure>
 
-    <para>Weitere Informationen finden Sie im Wiki unter
-      <ulink url="&url-wiki-de;Aktienhandel#Automatisierter_Kursabruf_mittels_cron">Automatisierter
-      Kursabruf mittels cron</ulink> und auf
+    <para>Weitere Informationen finden Sie im Wiki unter 
       <ulink url="&url-wiki-OQ;#Updating_enabled_Quotes_from_outside_GnuCash">Online Quotes Page on
```

Comment:
Gefällt dir die deutsche Seite nicht mehr oder ist das zuviel Arbeit, die synchron zu halten?

Reply:
na ja, auf der Seite steht zur Installation von F::Q nicht mehr so viel drin.

Reply:
Auch gut. Ein bischen stör ich mich noch an `Weitere Informationen`.
Es sollte eher in die Richtung gehen: Möglicherweise haben Benutzer im Wiki Ergänzungen und Aktualisierungen…